cms/apps/media/widgets.py

```diff
@@ -0,0 +1,24 @@
+from django import forms
+from django.utils.html import mark_safe
+from django.contrib.admin.widgets import ForeignKeyRawIdWidget
+
+class ImageThumbnailWidget(ForeignKeyRawIdWidget):
+    '''
+    A widget used to display a thumbnail image preview
+    '''
+    def render(self, name, value, attrs=None):
+        # Import here to avoid circular dependencies
+        from .models import File
+        output = []
+
+        if value:
+            file_obj = File.objects.filter(pk=value).first()
+
+            if file_obj:
+                image_url = file_obj.file.url
+                file_name = str(file_obj)
+                output.append(u' <a href="%s" target="_blank"><img src="%s" alt="%s" width="150" height="150"/></a> %s ' % \
+                            (image_url, image_url, file_name, 'Change:'))
```

Comment:
Is there a reason to format a string constant into a string, rather than it being part of the string?

Reply:
If I've understood your comment correctly, you meant why include 'Change' as a format parameter instead of just directly into the string? Not really any compelling reason to do this and I've now just added it to the string. .